README.md

```diff
@@ -21,13 +21,15 @@ You can find example implementations of this module in the tests folder (create_
 
 | Name | Version |
 |------|---------|
-| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 3.0 |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.15 |
+| <a name="requirement_aws"></a> [aws](#requirement\_aws) | >= 4.0 |
```

Comment:
Something went sideways here. There should only be one aws provider version requirement... 4.24.0...

Reply:
This was a result of having both top-level versions.tf and providers.tf files.  It's fixed now.